apps/e2e/cypress/e2e/FAPs.cy.ts

```diff
@@ -818,9 +847,6 @@ context('Fap reviews tests', () => {
 
       cy.login('officer');
 
-      cy.visit(`/FapPage/${createdFapId}?tab=1`);
-      cy.get('[data-cy="fap-reviewers-table"]').contains('0');
```

Comment:
I am wondering why this got removed

Reply:
It used to be that proposal totals were only populated for reviewers after the call ended, but I removed this filter as proposals can in a review status before then.  This part of the tests was testing that they were not appearing before call ends, I think. Saying that, I should remove the call ending in this tests as well.